audit_exceptions/throttled_formulae.json

```
@@ -7,4 +7,5 @@
   "gatsby-cli": 10,
   "quicktype": 10,
   "vim": 25
```

```suggestion
  "vim": 25,
```
Please squash your changes together into one commit with the commit message

    vim-gtk 8.2.2650 (new formula)

---

But I can't squash without having authorization to merge...

---

You don't need to merge your pull request to squash it. You just to do `git rebase -i HEAD~5`, squash the last four commits, and then `git push --force`.

---

I'm not on a machine with a git installation at the moment.
I'm basically doing everything on the website.
Is there another way?

---

Ok, let me fix it for you.